authority/authorize.go

```diff
@@ -386,8 +389,8 @@ func (a *Authority) AuthorizeRenewToken(ctx context.Context, ott string) (*x509.
 		return nil, errs.InternalServerErr(err, errs.WithMessage("error validating renew token"))
 	}
 
-	p, ok := a.provisioners.LoadByCertificate(leaf)
-	if !ok {
+	p, err := a.LoadProvisionerByCertificate(leaf)
```

Comment:
How will this work for certificates without the provisioner extension?

Reply:
We will need to change the issuer in `step` to use a fixed string similar to what I've done in `AuthorizeAdminToken`.